node/rpc/src/lib.rs

```diff
@@ -227,6 +236,9 @@ where
 
 	io.merge(Unique::new(client.clone()).into_rpc())?;
 
+	#[cfg(not(any(feature = "unique-runtime", feature = "quartz-runtime")))]
```

Comment:
По аналогии с другим местом использования
```suggestion
	#[cfg(feature = "opal-runtime")]
```

Reply:
@CertainLach, может сразу предустсмортеть будущее подклюбчение в друих рантаймах ? 
```
#[cfg(any(feature = "opal-runtime"))]
```